lib/ansible/plugins/connection/podman.py

```diff
@@ -87,6 +90,9 @@ def _podman(self, cmd, cmd_args=None, in_data=None):
                              stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
         stdout, stderr = p.communicate(input=in_data)
+        display.vvv("STDOUT %s" % stdout)
```

Comment:
I wonder if these should be more `v`s but not a blocker

Reply:
> I wonder if these should be more `v`s but not a blocker

thanks, changed to vvvvv

Reply:
fixed